pkg/config/settings.go

```diff
@@ -16,21 +16,59 @@
 
 package config
 
+import (
+	"fmt"
+	v1 "github.com/IBM/ibm-block-csi-operator/pkg/apis/csi/v1"
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"os"
+	"sigs.k8s.io/yaml"
+)
+
 const (
-	NodeDriverRegistrarImage    = "quay.io/k8scsi/csi-node-driver-registrar:v1.2.0"
-	CSIProvisionerImage         = "quay.io/k8scsi/csi-provisioner:v1.5.0"
-	CSIAttacherImage            = "quay.io/k8scsi/csi-attacher:v1.2.1"
-	CSILivenessProbeImage       = "quay.io/k8scsi/livenessprobe:v1.1.0"
-	CSISnapshotterImage      	= "quay.io/k8scsi/csi-snapshotter:v2.1.0"
-	CSIResizerImage             = "k8s.gcr.io/sig-storage/csi-resizer:v1.0.0"
-
-	ControllerTag = "1.5.0"
-	NodeTag       = "1.5.0"
-	NodeAgentTag  = "1.0.0"
-
-	DefaultNamespace = "default"
+	EnvNameCrYaml string = "CR_YAML"
+	NodeAgentTag         = "1.0.0"
+
 	DefaultLogLevel  = "DEBUG"
 	ControllerUserID = int64(9999)
 
 	NodeAgentPort = "10086"
+
+	IBMRegistryUsername    = "ibmcom"
+	K8SRegistryUsername    = "k8s.gcr.io/sig-storage"
+	QuayRegistryUsername   = "quay.io/k8scsi"
+	RedHatRegistryUsername = "registry.redhat.io/openshift4"
 )
+
+var DefaultCr v1.IBMBlockCSI
+
+var DefaultSidecarsByName map[string]v1.CSISidecar
```

Comment:
is it Default?

Reply:
yes, default per (operator+driver) version

Reply:
so what is not default?

Reply:
the sidecars running in the cluster